extensions/amp-consent/0.1/consent-policy-manager.js

```diff
@@ -76,14 +90,22 @@ export class ConsentPolicyManager {
     return isExperimentOn(win, 'multi-consent');
   }
 
+  /**
+   *
+   * @param {string} consentInstanceId
+   */
+  setLegacyConsentInstanceId(consentInstanceId) {
+    this.consentInstanceIdDepr_ = consentInstanceId;
+    this.init_();
+  }
+
   /**
    * Register the policy instance
    * Example policy config format:
    * {
    *   "waitFor": {
    *     "consentABC": [], // Can't support array now.
    *                       // All items will be treated as an empty array
```

Comment:
Yes I wanted to completely remove the support to `waitFor`. However, I can't do that now because it may break some of the existing usage. I'll create a follow up PR to support the new format and migrate later.